infra/Makefile.toml

```diff
@@ -0,0 +1,597 @@
+[config]
+default_to_workspace = false
+
+[env]
+CHAIN_NAME = { value = "root", condition = { env_not_set = ["CHAIN_NAME"] }}
+BALANCE = { value = "1000", condition = { env_not_set = ["BALANCE"] }}
+BASE_FEE = { value = "1000", condition = { env_not_set = ["BASE_FEE"] }}
+TIMESTAMP = { value = "1680101412", condition = { env_not_set = ["TIMESTAMP"] }}
+
+BASE_DIR="${HOME}/.ipc/${CHAIN_NAME}"
+
+BASE_TESTNET_DIR="${HOME}/.ipc-testnet"
+
+FM_DIR="${BASE_DIR}/fendermint"
+CMT_DIR="${BASE_DIR}/cometbft"
+
+GENESIS_FILE="${FM_DIR}/genesis.json"
+KEYS_DIR="${FM_DIR}/keys"
+KEY_NAME="validator_key"
+PUB_KEY_PATH="${KEYS_DIR}/${KEY_NAME}.pk"
+PRIV_KEY_PATH="${KEYS_DIR}/${KEY_NAME}.sk"
+
+NETWORK_NAME = "${CHAIN_NAME}"
+
+
+CMT_CONTAINER_NAME = "${NETWORK_NAME}-cometbft"
+FM_CONTAINER_NAME = "${NETWORK_NAME}-fendermint"
+ETHAPI_CONTAINER_NAME = "${NETWORK_NAME}-ethapi"
+
+CMT_DOCKER_IMAGE = "cometbft/cometbft:v0.37.x"
+FM_DOCKER_IMAGE = "fendermint:latest"
+TEST_DATA_DIR = "${CARGO_MAKE_WORKSPACE_WORKING_DIRECTORY}/fendermint/testing/smoke-test/test-data"
+TEST_SCRIPTS_DIR = "${CARGO_MAKE_WORKSPACE_WORKING_DIRECTORY}/fendermint/testing/smoke-test/scripts"
+ACTORS_BUNDLE = "${CARGO_MAKE_WORKSPACE_WORKING_DIRECTORY}/../builtin-actors/output/bundle.car"
+CMT_HOST_PORT = 26657
+ETHAPI_HOST_PORT = 8545
+# If this wasn't present, any wait task is skipped.
+CARGO_MAKE_WAIT_MILLISECONDS = 5000
+# This wait time seems to work locally.
+CMT_WAIT_MILLIS = 10000
+# Keep example logs to a minimum.
+VERBOSITY = ""
+
+[env.ci]
+CMT_WAIT_MILLIS = 20000
+# Help debug any issues with simplecoin by logging requests and responses.
+VERBOSITY = "--verbose"
+
+# infrastructure workflows:
+# cargo install cargo-make
+#
+# cd fendermint/testing/smoke-test
+# - then -
+# cargo make --profile ci
+# - or -
+# cargo make setup
+# cargo make test
+# docker logs smoke-ethapi
+# cargo make teardown
+
+[tasks.info]
+script="""
+echo
+echo Chain info:
+echo - Chain: ${CHAIN_NAME}
+echo - Balance: ${BALANCE}
+echo - Base Fee: ${BASE_FEE}
+echo - Timestamp: ${TIMESTAMP}
+echo
+echo Single node testnet layout:
+echo - IPC directory: ${BASE_DIR}
+echo - IPC Testnet directory: ${BASE_TESTNET_DIR}
+echo - CometBFT directory: ${CMT_DIR}
+echo - Fendermint directory: ${FM_DIR}
+echo - Keys directory: ${KEYS_DIR}
+echo - Genesis file: ${GENESIS_FILE}
+echo - Private key: ${PRIV_KEY_PATH}
+echo - Network: ${NETWORK_NAME}
+echo - CometBFT container: ${CMT_CONTAINER_NAME}
+echo - Fendermint container: ${FM_CONTAINER_NAME}
+echo
+echo
+echo 4 nodes testnet layout:
+echo - IPC directory: ${BASE_TESTNET_DIR}
```

Comment:
NB the docker-compose file doesn't respect this value.

Reply:
Fixed